plugin/evm/orderbook/hubbleutils/signed_orders.go

```diff
@@ -15,9 +15,10 @@ import (
 
 type SignedOrder struct {
 	LimitOrder
-	OrderType uint8    `json:"orderType"`
-	ExpireAt  *big.Int `json:"expireAt"`
-	Sig       []byte   `json:"sig"`
+	OrderType uint8       `json:"orderType"`
+	ExpireAt  *big.Int    `json:"expireAt"`
+	Sig       []byte      `json:"sig"`
+	OrderHash common.Hash `json:"orderHash"`
```

Comment:
due to reasons listed in the comment below, I think this is not ideal. Need to find another way to avoid recalculating the orderHash

Reply:
changed it now